sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/src/autorest.md

```diff
@@ -19,6 +19,8 @@ modelerfour:
 list-exception:
 - /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StreamAnalytics/streamingjobs/{jobName}/transformations/{transformationName}
 
+no-property-type-replacement:
+- ClusterInfo
```

Comment:
Does this has a nullable issue so that we add this configuration here? I did not see any configuration around the nullable issue on this. If not, please remove this
If this is blocking any test cases in any way, please feel free to add some comment above this to explain the reasons.

Reply:
Comments have been added
![image](https://user-images.githubusercontent.com/88413158/187165388-787cbf6d-5646-43b3-bbbe-46abc744fb71.png)